bindings/gumjs/gumdukvalue.c

```diff
@@ -701,6 +701,21 @@ _gum_duk_get_int64 (duk_context * ctx,
   }
 }
 
+static gboolean
+gum_duk_is_int64 (duk_context *ctx,
+                   duk_idx_t index,
+                   GumDukCore *core)
+{
+  duk_dup (ctx, index);
+  duk_push_heapptr (ctx, core->int64);
+
+  gboolean is_int64 = duk_instanceof (ctx, -2, -1);
```

Comment:
No declarations after statement; the declaration should be moved up, with a blank line following it, and the assignment kept here.